### PR TITLE
Bump mm dependency to release-5.23

### PR DESCRIPTION
**Pull request by agnivade**
#### Summary

Bump mm dependency to release-5.23

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-24355

### CODE REVIEW

**Comment by agnivade** [comment]
Wait, what's this now ? How is it failing with this PR ? :thinking:

**Comment by streamer45** [comment]
> Wait, what's this now ? How is it failing with this PR ?

It's the mutex I added to `model.Post`. Can't copy the struct like that. Should use `Clone()`.

**Comment by agnivade** [comment]
Ahh .. didn't realize this is related to server code. Thanks :+1: 

(terraform destruction going on now. So can't change the branch just yet. :P)

File: loadtest/store/memstore/random_test.go [hunk 1]
```diff
@@ -88,7 +88,7 @@ func TestRandomPost(t *testing.T) {
 func TestRandomPostForChannel(t *testing.T) {
 	s := New()
 	post, err := s.RandomPostForChannel("someId")
-	require.Empty(t, post)
+	require.Empty(t, post.Clone())
```
Comment (streamer45) [inline loadtest/store/memstore/random_test.go:91]: Maybe pass `&post` here.

Comment (agnivade) [reply loadtest/store/memstore/random_test.go]: done

**Pull request merged**